### DISCUSSION TITLE
Feature 실험 결과

### DISCUSSION THREAD

**Discussion by sooyounlee** [bcaitech1/p4-dkt-DKTJHGSD#79] [General]
| Feature | 넣기전 -> 넣은 후 성능 변화 | 함께 넣은 feature | 리더보드 제출 번호 ([리더보드 제출 결과](#12) 참조) | 
| :-------------: | :-------------: | :-------------: | :-------------: | 
| duration (문제 푸는 소요 시간) | 상승 (0.7233 -> 0.7286) | 기본 데이터 | 01 -> 12 | 
| duration (문제 푸는 소요 시간) 4분 이상 4분으로 치환 | 하락 (0.7233 -> 0.7225) | 기본 데이터 | 01 -> 14 |
| 소요시간 범주형 feature | NA | NA | NA |
| difficulty 범주형 feature | NA | NA | NA |
| (window 설정) 해당 태그 관련 문제 푼 수 | NA | NA | NA |
| (window 설정) 해당 시험지 관련 문제 푼 수 | NA | NA | NA |
| (window 설정) 이동 평균 (태그 별) | NA | NA | NA |
| (window 설정) 이동 평균 (시험지 별) | NA | NA | NA |
| (accumulation) 해당 태그 관련 문제 푼 수 | NA | NA | NA |
| (accumulation) 해당 시험지 관련 문제 푼 수 | NA | NA | NA |
| (accumulation) 이동 평균 (태그 별) | NA | NA | NA |
| (accumulation) 이동 평균 (시험지 별) | NA | NA | NA |
| 재접속 빈도 | NA | NA | NA |
| 요일 범주형 | NA | NA | NA |
| 주 범주형 | NA | NA | NA |
| 시간 범주형 | NA | NA | NA |

**Comment by joochanhyeong** [comment]
미리 틀 만들어주셔서 감사합니다!
